odd-platform-ui/src/redux/thunks/collectors.thunks.ts

```diff
@@ -0,0 +1,86 @@
+import {
+  CollectorApi,
+  Configuration,
+  CollectorList,
+  Collector,
+  CollectorApiGetCollectorsListRequest,
+  CollectorApiUpdateCollectorRequest,
+  CollectorApiRegisterCollectorRequest,
+  CollectorApiDeleteCollectorRequest,
+  CollectorApiRegenerateCollectorTokenRequest,
+} from 'generated-sources';
+import { createThunk } from 'redux/thunks/base.thunk';
+import { DeleteCollector } from 'redux/interfaces/collectors';
+import * as actions from 'redux/actions';
+import { BASE_PARAMS } from 'lib/constants';
+import { PaginatedResponse } from 'redux/interfaces/common';
+
+const apiClientConf = new Configuration(BASE_PARAMS);
+const apiClient = new CollectorApi(apiClientConf);
+
+export const fetchCollectorsList = createThunk<
+  CollectorApiGetCollectorsListRequest,
+  CollectorList,
+  PaginatedResponse<CollectorList>
+>(
+  (params: CollectorApiGetCollectorsListRequest) =>
+    apiClient.getCollectorsList(params),
+  actions.fetchCollectorsAction,
+  (
+    response: CollectorList,
+    request: CollectorApiGetCollectorsListRequest
+  ) => ({
+    ...response,
+    pageInfo: {
+      ...response.pageInfo,
+      page: request.page,
+      hasNext: request.size * request.page < response.pageInfo.total,
+    },
```

Comment:
As far as I know, now pagination comes from the backend correctly. I think now it is not necessary to consider hasNext in this way.

CC: @damirabdul

Reply:
done